information_for_beamlines_ds/__init__.py

```diff
@@ -12,8 +14,8 @@
 __author__ = "Michal Piekarski"
 __author_email__ = "michalpiekars@gmail.com"
 
-for package_name in find_packages():
-    package_import = __import__(package_name)
-    __doc__ += "%s: %s" % (package_name, package_import.__doc__)
-    __author__ += package_import.__author__ + ", "
-    __author_email__ += package_import.__author_email__ + ", "
+package_name=find_packages()
```

Comment:
My idea was to remove this part of `__init__.py` completely, not to remove the `for` statement only 😃